pkg/agent/baker_test.go

```diff
@@ -359,196 +366,206 @@ var _ = Describe("Assert generated customData and cseCmd", func() {
-				"--dynamic-config-dir":                "",
-			}
-		}, nil),
+		Entry("AKSUbuntu1804 with containerd and private ACR", "AKSUbuntu1804+Containerd+PrivateACR", "1.18.2",
+			func(config *datamodel.NodeBootstrappingConfiguration) {
+				config.ContainerService.Properties.AgentPoolProfiles[0].KubernetesConfig = &datamodel.KubernetesConfig{
+					ContainerRuntime: datamodel.Containerd,
+				}
 
-		Entry("AKSUbuntu1604 - dynamic-config-dir should always be removed", "AKSUbuntu1604+DynamicKubeletConfig", "1.16.13", func(config *datamodel.NodeBootstrappingConfiguration) {
-			config.KubeletConfig = map[string]string{
-				"--address":                           "0.0.0.0",
-				"--pod-manifest-path":                 "/etc/kubernetes/manifests",
-				"--cloud-provider":                    "azure",
-				"--cloud-config":                      "/etc/kubernetes/azure.json",
-				"--azure-container-registry-config":   "/etc/kubernetes/azure.json",
-				"--cluster-domain":                    "cluster.local",
-				"--cluster-dns":                       "10.0.0.10",
-				"--cgroups-per-qos":                   "true",
-				"--tls-cert-file":                     "/etc/kubernetes/certs/kubeletserver.crt",
-				"--tls-private-key-file":              "/etc/kubernetes/certs/kubeletserver.key",
-				"--tls-cipher-suites":                 "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_128_GCM_SHA256",
-				"--max-pods":                          "110",
-				"--node-status-update-frequency":      "10s",
-				"--image-gc-high-threshold":           "85",
-				"--image-gc-low-threshold":            "80",
-				"--event-qps":                         "0",
-				"--pod-max-pids":                      "-1",
-				"--enforce-node-allocatable":          "pods",
-				"--streaming-connection-idle-timeout": "4h0m0s",
-				"--rotate-certificates":               "true",
-				"--read-only-port":                    "10255",
-				"--protect-kernel-defaults":           "true",
-				"--resolv-conf":                       "/etc/resolv.conf",
-				"--anonymous-auth":                    "false",
-				"--client-ca-file":                    "/etc/kubernetes/certs/ca.crt",
-				"--authentication-token-webhook":      "true",
-				"--authorization-mode":                "Webhook",
-				"--eviction-hard":                     "memory.available<750Mi,nodefs.available<10%,nodefs.inodesFree<5%",
-				"--feature-gates":                     "RotateKubeletServerCertificate=true,a=b,PodPriority=true,x=y",
-				"--system-reserved":                   "cpu=2,memory=1Gi",
-				"--kube-reserved":                     "cpu=100m,memory=1638Mi",
-				"--dynamic-config-dir":                "",
-			}
-		}, nil),
+				config.KubeletConfig = map[string]string{}
```

Comment:
this diff is a wee bit confusing -- what actually changed here besides the //nolint?

I don't see anything at a glance but git seems to disagree

Reply:
specifically around ordering of test scenarios, the new line breaks make sense

Reply:
got it, all the func() have been shifted which is confusing. it looks fine in your branch